freqtrade/freqai/prediction_models/ReinforcementLearner.py

```diff
@@ -88,6 +88,33 @@ class MyRLEnv(Base5ActionRLEnv):
         User can override any function in BaseRLEnv and gym.Env. Here the user
         sets a custom reward based on profit and trade duration.
         """
+        def reset(self):
+
+            # Reset custom info
+            self.custom_info = {}
```

Comment:
I think we can put this in the `BaseEnvironment` so that it doesnt need to be defined for all use-cases.